.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
CodeQL complains about comments in JSON for the `tsconfig.json` file, but I'm not seeing any comments???

```bash
Analysis produced the following diagnostic data:

| Severity |                                                                           Message                                                                            |
+----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
| error    | Extraction failed in /home/runner/work/octoclairvoyant-webapp/octoclairvoyant-webapp/cypress/tsconfig.json with error Error: Comments are not legal in JSON. |
| error    | Extraction failed in /home/runner/work/octoclairvoyant-webapp/octoclairvoyant-webapp/cypress/tsconfig.json with error Error: Comments are not legal in JSON. |
| error    | Extraction failed in /home/runner/work/octoclairvoyant-webapp/octoclairvoyant-webapp/cypress/tsconfig.json with error Error: Comments are not legal in JSON. |
| none     | (61 results for diagnostic "Successfully extracted files")                                                                                                   |
Analysis produced the following metric data:

|                                   Metric                                   | Value |
+----------------------------------------------------------------------------+-------+
| Total lines of user written JavaScript and TypeScript code in the database |  2489 |
| Total lines of JavaScript and TypeScript code in the database              |  2551 |

/opt/hostedtoolcache/CodeQL/0.0.0-20211025/x64/codeql/codeql database print-baseline /home/runner/work/_temp/codeql_databases/javascript
Counted a baseline of 2552 lines of code for javascript.
Counted a baseline of 2552 lines of code for javascript.
```

Reply:
It's complaining about the `tsconfig.json` within cypress folder, I see a few comments there: https://github.com/octoclairvoyant/octoclairvoyant-webapp/blob/main/cypress/tsconfig.json

Reply:
Ooh whoops... I was looking in the wrong file!